x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -85,11 +93,20 @@ export class TaskRunnerFactory {
           },
         };
 
-        const executorResult = await actionExecutor.execute({
-          params,
-          actionId,
-          request: fakeRequest,
-        });
+        let executorResult: ActionTypeExecutorResult;
+        try {
+          executorResult = await actionExecutor.execute({
```

Comment:
I think we'll need a similar try/catch around the http route for action execution - or maybe that's in a different PR - https://github.com/elastic/kibana/blob/b56cd414127382b9d8928ec328d83df462ee2575/x-pack/plugins/actions/server/routes/execute.ts#L52-L56

Reply:
Yeah, this was handled within the previous "Add license checks to action HTTP APIs" PR here: https://github.com/elastic/kibana/pull/59153/files#diff-4c51aa8615e4e2fb384b4a72f14e6a21R63-R65.